util/Fs.cpp

```diff
@@ -252,7 +252,9 @@ ResourcePressure Fs::readRespressure(const std::string& path) {
     // aggr 316016073
     // some 0.00 0.03 0.05
     // full 0.00 0.03 0.05
-    OCHECK(lines.size() == 3);
+    if (lines.size() != 3) {
+      throw std::runtime_error("could not read Mempressure process terminated");
```

Comment:
I think this would be more informative, as there are usually many processes in a cgroup:
```suggestion
      throw std::runtime_error("cgroup is no longer valid: " + path);
```